applications/CoSimulationApplication/python_scripts/base_classes/co_simulation_solver_wrapper.py

```diff
@@ -0,0 +1,137 @@
+from __future__ import print_function, absolute_import, division  # makes these scripts backward compatible with python 2.6 and 2.7
+
+# CoSimulation imports
+import KratosMultiphysics.CoSimulationApplication.factories.io_factory as io_factory
+import KratosMultiphysics.CoSimulationApplication.co_simulation_tools as cs_tools
+from KratosMultiphysics.CoSimulationApplication.coupling_interface_data import CouplingInterfaceData
```

Comment:
I've had problems importing pythons like this (with the whole path) as cx_freeze do not seem to find it when doing the packaging..

Reply:
I found something interesting about this, so I will create a separated issue. Sorry for the noise

Reply:
I know, @pooyan-dadvand told me abt this
If you open an issue then we can discuss there